src/main/java/bio/terra/service/resourcemanagement/MetadataDataAccessUtils.java

```diff
@@ -103,7 +103,7 @@ public AccessInfoModel accessInfoFromSnapshot(
       return makeAccessInfoAzure(
           snapshot, storageAccountResource, snapshot.getTables(), profileModel, userRequest);
     } else {
-      throw new IllegalArgumentException("Unrecognized cloud platform");
+      throw new IllegalArgumentException();
```

Comment:
what's the reasoning behind this change?

Reply:
consistency.  The no arg constructor calls the string constructor with that value.  Seemed better to just call that (it was a drive by)

Reply:
Wait. I'm confused. Why are we removing information that would be helpful to the user?

Reply:
🤦‍♂️ sorry...I meant to convert this to a `InvalidCloudPlatformException` exception. Will update